### PR TITLE
Update mycroft.conf

### PR DESCRIPTION
**Pull request by JarbasAl**

### CODE REVIEW

File: buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf [hunk 1]
```diff
@@ -4,15 +4,10 @@
...
   "ipc_path": "/dev/shm/mycroft/ipc/",
-  "enclosure": {
-    "platform": "OpenVoiceOS",
-    "ntp_sync_on_boot": true
-  },
   "gui": {
     "extension": "smartspeaker",
     "idle_display_skill": "skill-ovos-homescreen.openvoiceos"
   },
-  "disable_xdg": false,
```
Comment (JarbasAl) [inline buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf:15]: from HolmesV times, does not exist in ovos

File: buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf [hunk 2]
```diff
@@ -4,15 +4,10 @@
   "play_wav_cmdline": "paplay %1",
   "play_mp3_cmdline": "mpg123 %1",
   "ipc_path": "/dev/shm/mycroft/ipc/",
-  "enclosure": {
```
Comment (JarbasAl) [inline buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf:7]: these are default values already, enclosure is deprecated in ovos

File: buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf [hunk 3]
```diff
@@ -4,45 +4,17 @@
...
   "gui": {
     "extension": "smartspeaker",
     "idle_display_skill": "skill-ovos-homescreen.openvoiceos"
   },
-  "disable_xdg": false,
   "listener": {
     "mute_during_output": false,
     "instant_listen": true
   },
-  "hotwords": {
```
Comment (JarbasAl) [inline buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf:20]: these are default values already

File: buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf [hunk 4]
```diff
@@ -4,44 +4,20 @@
   "play_wav_cmdline": "paplay %1",
   "play_mp3_cmdline": "mpg123 %1",
   "ipc_path": "/dev/shm/mycroft/ipc/",
-  "enclosure": {
-    "platform": "OpenVoiceOS",
-    "ntp_sync_on_boot": true
```
Comment (j1nx) [inline buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf:9]: Does it take care of ntp sync by default?

Comment (JarbasAl) [reply buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf:9]: oops, no it doesnt

Comment (j1nx) [reply buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf:9]: Hmm, strange that I then didn't notice it. I am not starting the enclosure service anymore, so either way this should have not been used.

Let me double check.

Comment (JarbasAl) [reply buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf:9]: its used by skills service DevicePrimer

Comment (j1nx) [reply buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf:9]: So, am I correct then to better leave this for now?

**Comment by j1nx** [comment]
Going to merge this and leave the enclosure block in for a little while.

The enclosure string is also currently used to be sent to the remote backend if a user follows the online mycroft route during first boot.

**Pull request merged**